Clarification on uniqueness of the CoRIM ID (#474)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

### Identity {#sec-corim-id}

> A CoRIM Identifier uniquely identifies a CoRIM instance within the context of a CoRIM issuer.  In other words the CoRIM identifier is not globally unique,

Reviewer: Is `CoRIM issuer` synonymous with `signer-name` and/or `kid` within the CoRIM?

> A CoRIM Identifier uniquely identifies a CoRIM instance within the context of a CoRIM issuer.  In other words the CoRIM identifier is not globally unique,

Reviewer 2: The catch is that when CoRIM ID is using `uuid` construct it will be globally unique, hence I would say may not be, instead of an affirmative, is not

> A CoRIM Identifier uniquely identifies a CoRIM instance within the context of a CoRIM issuer.  In other words the CoRIM identifier is not globally unique,

Reviewer 3: If we are going down this route, alternatively:

```suggestion
A CoRIM Identifier uniquely identifies a CoRIM instance within the context of a CoRIM issuer.
In essence, the CoRIM identifier is intended to be globally unique,
```

> but can be used to distinguish CoRIMs that come from the same issuer.

Reviewer 2: ```suggestion
but it is used to distinguish CoRIMs that come from the same issuer.
```

> ### Identity {#sec-corim-id}

Reviewer 2: ```suggestion
### CoRIM Identifier {#sec-corim-id}
```
